packages/shared/components/Sidebar.svelte

```diff
@@ -19,13 +19,47 @@
     let prevPartiallyUnstakedAmount = 0 // store the previous unstaked funds to avoid notifying when unstaked funds decrease
     let showStakingNotification = false
 
-    const hasTitleBar = document.body.classList.contains('platform-win32')
     const profileColor = 'blue' // TODO: each profile has a different color
 
     $: profileInitial = getInitials($activeProfile?.name, 1)
     $: healthStatus = $networkStatus.health ?? 0
     $: $dashboardRoute, $stakingEventState, $partiallyUnstakedAmount, manageUnstakedAmountNotification()
 
+    $: $dashboardRoute, $activeProfile?.hasVisitedStaking || showStakingNotification, updateSidebar()
+
+    let sidebarTabs = [
```

Comment:
Maybe it makes sense to define a SideBarTab type here?

Reply:
Then it also makes it convenient to only pass one prop instead of 5 to SideBarTab

Reply:
updated